server/src/controller/ReviewsRouter.js

```diff
@@ -0,0 +1,31 @@
+const express = require('express');
+const router = express.Router();
+const reviewsService = require('../service/ReviewsService');
+
+// CREATE
+// New Review Post
+router.post('/reviewPost', async (req, res) => {
+    //add username to body through the jwt tokens username
+    const data = await reviewsService.createNewReview(req.body);
+
+    if(data === 'Please fill out fields correctly') {
```

Comment:
Instead of setting data to a String, we should have `createNewReview` throw an error.